docs/guides/job-limits.mdx

```diff
@@ -21,6 +21,49 @@ At most, **10 million** executions are allowed.  The number of executions is the
 
 For example, if you have a PUB with one circuit, observables with shape (1, 6), and parameters with shape (4, 1), this would render $4 \times 6 = 24$ circuits (or fewer, if some observables commute). If you requested 2,000 shots, then the total number of executions is $24 \times 2,000 = 48,000$.
 
+When the observables commute, the number of executions can be further reduced. Generally, we calculate number of observables that commute by using Qiskit as follows:
+
+```python
+number_of_commuting_groups = observables.group_commuting(qubit_wise=True)
+```
+OR
+```python
+number_of_commuting_groups = observables.group_qubit_wise_commuting()
+```
+
+After determining the number of commuting groups, the total execution is given by:
+`Total executions = number_of_commuting_groups * number_of_parameter_set * number_of_shots`
```

Comment:
As commented [here](https://github.com/Qiskit/documentation/issues/3836#issuecomment-3598353334), this limit now only applies to Sampler jobs, so this section is no longer needed (as Sampler doesn't take observables).